### PR TITLE
Add a link to chat room on gitter

### PR DESCRIPTION
**Pull request by malarzm**
Also I've removed IRC as we're abandoning the ship and moved SO up so it looks more "preferred"

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 [![Build Status](https://secure.travis-ci.org/doctrine/mongodb-odm.png?branch=master)](http://travis-ci.org/doctrine/mongodb-odm)
 [![Code Coverage](https://scrutinizer-ci.com/g/doctrine/mongodb-odm/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/doctrine/mongodb-odm/?branch=master)
+[![Gitter](https://badges.gitter.im/doctrine/mongodb-odm.svg)](https://gitter.im/doctrine/mongodb-odm?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```
Comment (alcaeus) [inline README.markdown]: Can we leave out the UTM tracking stuff? 😇

Comment (malarzm) [reply README.markdown]: Sure, I thought I got rid of it after pasting ;)

File: docs/en/index.rst [hunk 2]
```diff
@@ -10,10 +10,10 @@ Getting Help
 If this documentation is not helping to answer questions you have about
 Doctrine MongoDB ODM don't panic. You can get help from different sources:
 
+-  Gitter chat room `#doctrine/mongodb-odm <https://gitter.im/doctrine/mongodb-odm>`_
+-  On `StackOverflow <http://stackoverflow.com/questions/tagged/doctrine-odm>`_
```
Comment (jmikola) [inline docs/en/index.rst]: Small nit, but they're branded name is actually two words: "Stack Overflow"

Comment (jmikola) [reply docs/en/index.rst]: https://github.com/doctrine/mongodb-odm/pull/1695

**Pull request merged**